application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -635,44 +635,36 @@ define('diagram-store', ['jquery', 'xwiki-meta', 'xwiki-utils', 'diagram-utils',
   };
 
   var getTagName = function(element) {
-    if (!element) { return ''; }
+    if (!element || isTextNode(element)) { return ''; }
     return element.tagName.toLowerCase();
```

Comment:
If it's not a text node then it can be a comment node or an attribute node, or a document fragment, so not necessarily an element node. I would simply do:

```suggestion
    return element && element.tagName && element.tagName.toLowerCase();
```